docs/gauge_agg.md

```diff
@@ -0,0 +1,76 @@
+# Gauge Aggregates [<sup><mark>experimental</mark></sup>](/docs/README.md#tag-notes)
+
+A gauge is a metric similar to a counter, with the primary difference being
+that it measures a value that varies up and down over time, rather than an
+ever-increasing COUNT of the number of times something happened.
+Examples include resource utilization metrics, precipitation levels,
+or temperatures.
+
+`gauge_agg` currently shares implementation with `counter_agg` but without the
+resetting logic.  This means it enforces ordering even though that is not
+necessarily required for all gauge aggregates.  We may offer an additional
+unordered gauge aggregate in the future.
+
+# Test table
+
+Examples below are tested against the following table:
+
+```SQL ,non-transactional
+SET TIME ZONE 'UTC';
+CREATE TABLE gauge_test (
+    measure_id      BIGINT,
```

Comment:
Maybe we can use a real example with some more representative names than the testing one? If we develop the example above about diffs between gauges and counters that might help. I just find sometimes the concreteness can help with understanding.